scripts/xpcs_corr_gpu_client.py

```diff
@@ -0,0 +1,109 @@
+#!/home/beams/8IDIUSER/.conda/envs/gladier/bin/python
+
+## /home/beams/8IDIUSER/.conda/envs/gladier/bin/python /home/beams10/8IDIUSER/DM_Workflows/xpcs8/automate/raf/gladier-xpcs/scripts/xpcs_corr_client.py --hdf '/data/xpcs8/2019-1/comm201901/cluster_results/A001_Aerogel_1mm_att6_Lq0_001_0001-1000.hdf' --imm /data/xpcs8/2019-1/comm201901/A001_Aerogel_1mm_att6_Lq0_001/A001_Aerogel_1mm_att6_Lq0_001_00001-01000.imm --group 0bbe98ef-de8f-11eb-9e93-3db9c47b68ba
+
+# Enable Gladier Logging
+#import gladier.tests
+
+import argparse
+import os
+import pathlib
+
+from gladier_xpcs.flow_gpu import XPCSGPUFlow
+from gladier_xpcs.deployments import deployment_map
+
+
+def arg_parse():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('--hdf', help='Path to the hdf (metadata) file',
+                        default='/data/xpcs8/2019-1/comm201901/cluster_results/'
+                                'A001_Aerogel_1mm_att6_Lq0_001_0001-1000.hdf')
+    parser.add_argument('--raw', help='Path to the raw data file. Multiple formats (.imm, .bin, etc) supported',
+                        default='/data/xpcs8/2019-1/comm201901/A001_Aerogel_1mm_att6_Lq0_001'
+                                '/A001_Aerogel_1mm_att6_Lq0_001_00001-01000.imm')
+    parser.add_argument('--qmap', help='Path to the qmap file',
+                        default='/data/xpcs8/partitionMapLibrary/2019-1/comm201901_qmap_aerogel_Lq0.h5')
+    parser.add_argument('--group', help='Visibility in Search', default=None)
+    parser.add_argument('--deployment','-d', default='hannah-polaris', help=f'Deployment configs. Available: {list(deployment_map.keys())}')
+    parser.add_argument('--corr_gpu_loc', default='/eagle/projects/APSDataAnalysis/XPCS/mchu/xpcs_boost/gpu_corr.py',
```

Comment:
We need to investigate this path. This expose the `project` path which we are trying to hide.
It also should go out of `mchu` into either this package or a separate package. The code itself does not have any other python dependency.